sormas-backend/doc/UserDataAccess.md

```diff
@@ -1,62 +1,131 @@
-# Basics
+# User data access
+
+## Basics
 In general data access & synchronisation is based on the following rules:
 
 * hospital informant: all data associated to the facility (cases)
 * community informant: all data associated to the community (cases)
 * officers: all data associated to the district (cases, events)
 * supervisors: all data associated to the region
+* observers: all data associated to their jurisdiction (nation, region or district)
+* port of entry users: all data associated to their jurisdiction (nation, region or district)
 * all data that is created by the user
-* all data associated to the above data (contacts, tasks, persons, visits)
+* all data associated to the data created by the user (contacts, tasks, persons, visits)
+* port health users can only see port health cases, no matter the user jurisdiction
 * special case: if an officer has access to a task or contact whose case/event/contact is not available, the association link should be inactive
-* user rights (e.g. allowed to see cases) are documented in the [user rights table](sormas-api/src/main/resources/doc/SORMAS_User_Rights.xlsx)
+* user rights (e.g. allow seeing cases) and mapping of user roles to a jurisdiction level is documented in the [user rights table](https://github.com/hzi-braunschweig/SORMAS-Project/tree/development/sormas-api/src/main/resources/doc/SORMAS_User_Rights.xlsx)
 
-# Case
+## Case
 * only users that are permitted to see cases at all
 * whoever created the case or is assigned to it is allowed to access it
-* access by region/district/community/health facility/point of entry of the case
+* all cases accessible for National users
+* access by jurisdiction, that is based region/district/community/health facility/point of entry of the case
 * cases that are made public can be accessed by all users permitted to see cases (this feature can be disabled)
+* edit is possible only for cases that are part of the users jurisdiction
+* view of personal and sensitive data is possible only for cases that are part of the user's jurisdiction
 
-# Contact
+## Contact
 * only users that are permitted to see contacts at all
 * whoever created it or is assigned to it is allowed to access it
+* all contacts accessible for National users
 * users see all contacts of their cases
-* access by region/district of the contact
+* access by jurisdiction, that is using region/district of the contact and jurisdiction of the case the contact is assigned to
```

Comment:
see above;
also, this should be a bit clearer:

access by jurisdiction that is using the region/district of the contact or, if not available, the jurisdiction of the source case of the contact